starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,90 @@
+*, body, html {
+    margin: 0;
+    padding: 0;
+    font-size: 16px;
+    font-family: 'Work Sans', Helvetica, Arial, sans-serif;
+}
+
+body {
+    padding-bottom: 100px;
+}
+
+h1 {
+    color: #fff;
+    font-size: 3rem;
+    margin: auto;
+}
+
+h2 {
+    font-size: 1.2rem;
+    margin-bottom: 25px;
+}
+
+section {
+    width: 70%;
+    margin: 0 auto;
+    padding: 45px 0;
+    border-bottom: 1px solid #999;
+}
+
+section:last-child {
+    border-bottom: none;
+}
+
+img {
+    display: block;
+    width: 50%;
+    margin: 45px auto 0;
+}
+
+ul {
+    list-style-type: none;
+    margin-left: 20px;
+    font-size: 1.2rem;
+}
+
+ul li {
+    position: relative;
+    margin-bottom: 3px;
+}
+
+#hero {
+    background-image: url(../images/apple-pie.jpg);
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin-top: 0;
+    margin-bottom: 45px;
+    padding: 30% 0;
+    text-align: center;
+}
+
+#description {
+    font-size: 1.2rem;
+}
+
+#ingredients ul li:before {
+    content: "-";
```

Comment:
Smart stuff! Maybe someday they'll add a dash setting to the list-style property. Hopefully 🤞